project.clj

```diff
@@ -39,9 +50,10 @@
                  [colorize "0.1.1" :exclusions [org.clojure/clojure]] ; string output with ANSI color codes (for logging)
                  [com.amazon.redshift/redshift-jdbc41 "1.2.8.1005"]   ; Redshift JDBC driver
                  [com.cemerick/friend "0.2.3"                         ; auth library
-                  :exclusions [commons-codec
-                               org.apache.httpcomponents/httpclient
+                  :exclusions [com.google.inject/guice
```

Comment:
just curious, what's the reason behind this change?

Reply:
I was able to remove it